src/pages/SignInPage/SignInPage.jsx

```diff
@@ -426,7 +425,7 @@ const SignInPage = () => {
               }
             />
           </div>
-          {numeroDocumentoError ? (
+          {documentNumberError ? (
             <span className="sign-in-page-form-text-error-id sign-in-page-form-text-error">
```

Comment:
The conditional rendering has a very good intention, but I don't feel comfortable about the else condition, why not a react fragment, i.e.
```js
{variable ? (
  <p>
    Text
  </p>
  ) : (
  <></>
)}
```
or else:
```js
<span>
{variable ? {languageSelector(language, 'signInIdNumberError')} : ""}
</span>
```
This would considered to each case in this PR where you are applying that, else if, let's discuss it.